integr8ly/integreatly-operator
# MGDAPI-5728 refactor alert manager secret creation

# Issue link
https://issues.redhat.com/browse/MGDAPI-5728

# What
- update to the reconciliation of alert manager secret. Secret has been moved to rhoam ns and reconciliation is no longer triggered by OO but the bootstrap stage.

# Verification steps
- Install and uninstall RHOAM from this branch and confirm it works as expected. 
(ensure to run make cluster/prepare/local first to create monitoring stack)
